core/src/main/java/com/homeaway/streamplatform/streamregistry/streams/ManagedKStreams.java

```diff
@@ -98,4 +106,22 @@ public void stop() {
         Preconditions.checkState(view != null, "ManagedKStreams not started. Please start before using.");
         return view.all();
     }
+
+
+    private Materialized createMaterialized(SchemaManagerConfig schemaManagerConfig){
+        final Map<String, String> serdeConfig =
+            Collections.singletonMap(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG,
+                schemaManagerConfig.getProperties().get("schema.registry.url").toString());
```

Comment:
You can reuse `AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG` instead of `"schema.registry.url"`

Do you need `.toString()`? The `.get` should return Object, so you would usually cast that, not `toString`. Doesn't hurt to do a null-check if needed, as well, or `Optional.ofNullable(schemaManagerConfig.getProperties().get("...")).orElseThrow()`